kong/plugins/grpc-gateway/kong-plugin-grpc-gateway-0.1.3-1.rockspec

```diff
@@ -0,0 +1,32 @@
+package = "kong-plugin-grpc-gateway"
```

Comment:
Is this rockspec being picked up? If not we can remove it (and pin the dependency
below in Kong's rockspec, which there's already pinneed, so we can just remove this
file)

Reply:
as i remember, we agreed to preserve the individual plugin's rockspecs, for the (remote) possibility of installing them separately.  of course, none of the distribution images would use LuaRocks for that.